servers/audio_server.cpp

```diff
@@ -404,8 +404,22 @@ void AudioServer::_mix_step() {
 			buf[i] = playback->lookahead[i];
 		}
 
-		// Mix the audio stream.
-		unsigned int mixed_frames = playback->stream_playback->mix(&buf[LOOKAHEAD_BUFFER_SIZE], playback->pitch_scale.get(), buffer_size);
+		unsigned int mixed_frames = 0;
+
+		// For audio scheduled to start later, fill the buffer with silence frames.
+		uint64_t scheduled_start_frame = playback->scheduled_start_frame.get();
+		if (scheduled_start_frame > mix_frames) {
+			unsigned int silence_frames = MIN(scheduled_start_frame - mix_frames, buffer_size);
```

Comment:
```suggestion
			unsigned int silence_frames = MIN(scheduled_start_frame - mix_frames, buffer_size);
```

Should frames be 64 bit?

Reply:
Done, though it probably wouldn't have caused any issues since `buffer_size` is 32 bit. Makes sense to keep the type consistent however.